KoreBuild-dotnet/build/_xunit-test.shade

```diff
@@ -51,6 +51,9 @@ projectFile=''
                         testArgs = " -parallel none";
                     }
 
+                    // Removing app domains to allow custom test discovery
+                    testArgs += " -noappdomain";
```

Comment:
I have been trying to come up with a scenario with binding redirect that will be broken by this change and so far it seems to work. I have also checked a few repos, such as Security, to see if this change will break them.